javascript/packages/orchestrator/src/chainSpec.ts

```diff
@@ -130,14 +130,12 @@ export async function addBalances(specPath: string, nodes: Node[]) {
         const balanceToAdd = stakingBond
           ? node.validator && node.balance > stakingBond
             ? node.balance
-            : stakingBond! + BigInt(1)
+            : stakingBond! * BigInt(2) // Double the balance we use for stake
```

Comment:
we only need to add ED to staking bond, but this is also fine since staking bond should always be greater than ED.

Reply:
Great. Yes, set to the double is an easy fix.